README.rst

```diff
@@ -292,6 +292,17 @@ simple ``str`` in the default encoding as they will be implicitly
 decoded to text. Some backends may accept ``bytes`` for these parameters,
 but such usage is discouraged.
 
+
+Exceptions
+==========
+
+The keyring lib raises following exceptions:
+
+* ``keyring.errors.KeyringError``: Base Error class for all exceptions in keyring lib.
+* ``keyring.errors.InitError``: Raised when the password can't be initialized.
```

Comment:
Looks like a typo here, it should be "the keyring can't be initialized" I suppose.